src/apps/Altinn.Authorization.DeployApi/src/Altinn.Authorization.DeployApi/Program.cs

```diff
@@ -0,0 +1,30 @@
+using Altinn.Authorization.DeployApi.BootstrapDatabase;
+using Azure.Core;
+using Azure.Identity;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+
+var builder = WebApplication.CreateBuilder(args);
+
+TokenCredential cred;
+if (builder.Environment.IsDevelopment())
+{
+    cred = new DefaultAzureCredential();
+}
+else
+{
+    cred = new ManagedIdentityCredential();
```

Comment:
I do believe that `ManagedIdentityCredential` defaults to using system assigned identity if not specified a client_id, we are using a user assigned identity, just need to inject it.

We can also make an exception for this service to use SystemAssigned as this service doesn't depends on external conn during boot

Reply:
Added config for client id.